packages/client/hmi-client/src/components/navbar/tera-notification-panel.vue

```diff
@@ -32,10 +32,22 @@
 					{{ getTitleText(item) }}
 					<tera-asset-link :label="item.assetName" :asset-route="getAssetRoute(item)" />
 				</p>
-				<p class="msg">{{ item.msg }}</p>
+				<div class="notification-path-text msg">
+					<p>Project: {{ getProjectName() }}</p>
+					<p v-if="item.status !== 'Completed'">{{ item.msg }}</p>
```

Comment:
are those status part of an enum type somewhere.
Also, it would be nice to have function like `isComplete` to do those test.

Reply:
added a function &`enum to test the status